app/forms/state_file/az_state_credits_form.rb

```diff
@@ -0,0 +1,15 @@
+module StateFile
+  class AzStateCreditsForm < QuestionsForm
+    set_attributes_for :intake, :tribal_member, :tribal_wages, :armed_forces_member, :armed_forces_wages
+
+    validates :tribal_wages, presence: true, allow_blank: false, if: -> { tribal_member == "yes" }
```

Comment:
enums when configured with a `prefix` create methods like `tribal_member_yes?` which we generally prefer to `tribal_member == "yes"`

Reply:
i don't think it's an enum yet here though? it doesn't seem to recognize those methods on validation